windows/bazel/get_workspace_status

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+# This file was imported from https://github.com/bazelbuild/bazel at d6fec93.
+
+# This script will be run bazel when building process starts to
+# generate key-value information that represents the status of the
+# workspace. The output should be like
+#
+# KEY1 VALUE1
+# KEY2 VALUE2
+#
+# If the script exits with non-zero code, it's considered as a failure
+# and the output will be discarded.
+
+# For Envoy in particular, we want to force binaries to relink when the Git
+# SHA changes (https://github.com/envoyproxy/envoy/issues/2551). This can be
+# done by prefixing keys with "STABLE_". To avoid breaking compatibility with
+# other status scripts, this one still echos the non-stable ("volatile") names.
+
+# If this SOURCE_VERSION file exists then it must have been placed here by a
+# distribution doing a non-git, source build.
+# Distributions would be expected to echo the commit/tag as BUILD_SCM_REVISION
+if [ -f SOURCE_VERSION ]
+then
+    echo "BUILD_SCM_REVISION $(cat SOURCE_VERSION)"
+    echo "STABLE_BUILD_SCM_REVISION $(cat SOURCE_VERSION)"
```

Comment:
`BUILD_SCM_VERSION` is a known volatile key:

https://github.com/bazelbuild/bazel/blob/4304542a7438acaeb09559e71a971e56fbecc4e5/src/main/java/com/google/devtools/build/lib/bazel/BazelWorkspaceStatusModule.java#L355-L359

What's the reason to export the same info as a stable key?

Reply:
This script was copied from here: https://github.com/envoyproxy/envoy/blob/master/bazel/get_workspace_status

If we understand correctly, these variables are used for linkstamping: https://github.com/envoyproxy/envoy/blob/998b8119ed57269e77d0a03b1b92e7a98c5d30d1/source/common/common/version_linkstamp.cc

Since we haven't gotten linkstamping to work on Windows, we will just remove this